groups/models.py

```diff
@@ -161,6 +167,9 @@ def delete_state(self):
     def is_deleted(self):
         return self.state == self.STATE_DELETED
 
+    def __str__(self):
+        return 'Comment by {}'.format(self.user.get_full_name())
```

Comment:
what the fuck

`vars(comment)` contains `discussion_id: 84`

The error still happens in the very next line.

Is `discussion_id` some kind of special attribute that isn't exposed for normal access?